src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```

Comment:
Missing header comment. All non trivial classes/methods should have javadoc format header comments.